src/CodeStyle/Core/CodeFixes/FormattingCodeFixHelper.cs

```diff
@@ -2,16 +2,12 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-#if CODE_STYLE
-extern alias CodeStyle;
```

Comment:
📝 Changing this is out of scope for this pull request

Reply:
This was done to completely avoid extern aliases in CodeStyle layer. I can revert this and we can take this up as a separate work, which needs approval in the design meeting before implementing.

Reply:
As mentioned in previous comment on Formatter.cs, this is a required change as extern aliases are not supported in VB.